src/VMware/custom/LegalTerm.txt

```diff
@@ -0,0 +1,6 @@
+LEGAL TERMS
+Azure VMware Solution is an Azure Service licensed to you as part of your Azure subscription and subject to the terms and conditions of the agreement under which you obtained your Azure subscription (https://azure.microsoft.com/support/legal/). The following additional terms also apply to your use of AVS:
```

Comment:
Legal term should define the meaning of AVS explicitly. One simple way is "Azure VMware Solution (AVS) is ......"